src/lib/shared/api/adapters.ts

```diff
@@ -24,10 +24,23 @@ export interface SBEpisode {
 }
 
 export interface SBTranscript {
+  id: number;
   episode_id: SBEpisode['id'];
   content: string;
 }
 
+export interface SBFavourite {
```

Comment:
I, for one, would almost always prefer the British spelling, except that we already used `favorite` everywhere else in the project, and I value consistency more)

Reply:
I have very flaky standards when it comes to British vs English, because I like to say _favorite_ yet _neighbour_, but in this case I think it was Magomed who named the table in Supabase, and I thought that we should keep the Supabase objects with a `u`, but the rest of the code may proceed without it. Because of that, I don't really want to change this to be without the `u`